Dockerfile

```diff
@@ -791,7 +791,7 @@ COPY --from=pkg-grub-arm64 /usr/lib/grub /usr/lib/grub
 COPY --from=pkg-grub-amd64 /usr/lib/grub /usr/lib/grub
 COPY --from=unicode-pf2 /usr/share/grub/unicode.pf2 /usr/share/grub/unicode.pf2
 RUN ln /bin/installer /bin/imager
-RUN find /bin /etc /lib /usr /sbin | grep -Ev '/etc/hosts|/etc/resolv.conf' \
+RUN find /bin /etc /lib /usr /sbin | grep -Ev '/etc/hosts|/etc/resolv.conf|/etc/resolv-global.conf' \
```

Comment:
it doesn't have to be there (these are not Talos files)

Reply:
Removed.